src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -224,10 +228,9 @@ class MsalAuthorizeService implements AuthorizeService {
     async completeSignIn() {
         // Make sure that the redirect handler has completed execution before
         // completing sign in.
-        await this._redirectCallback;
-        const account = this.getAccount();
-        if (account) {
-            return this.success(account);
+        var state = await this._redirectCallback;
+        if (state) {
+            return this.success(state.state);
```

Comment:
What's the reasoning for the account -> state change here?

Reply:
The `account` parameter doesn't actually store the `returnUrl` property that is needed to correctly redirect.